### PR TITLE
PS-3454 update to python v10

### PR DESCRIPTION
**Pull request by pivnicek**
https://keboola.atlassian.net/browse/PS-3454

### CODE REVIEW

**Comment by pivnicek** [comment]
The build is working, but I'm going to rebase it so it will need to run again

File: Dockerfile [hunk 1]
```diff
@@ -7,15 +7,15 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
...
     && apt-get update && ACCEPT_EULA=Y apt-get install -y --no-install-recommends \
         libgeos-c1v5 \
-        python-numpy \
-        python-scipy \
-        python-matplotlib \
-        ipython \
+        python3-numpy \
+        python3-scipy \
+        python3-matplotlib \
+        python3-ipython \
```
Comment (pivnicek) [inline Dockerfile]: for some unknown reason I had to do this :-)

File: Dockerfile [hunk 2]
```diff
@@ -105,20 +105,27 @@ RUN mkdir /home/default \
...
         tqdm \
         vincent \
+        wheel \
         xgboost \
         xlrd \
 	&& chown :users -R /home/default \
 	&& chmod a+rwx -R /home/default
 
-RUN pip3 install --no-cache-dir --upgrade --force-reinstall git+git://github.com/keboola/python-docker-application.git@2.2.0 \
+RUN pip3 install --no-cache-dir --upgrade --force-reinstall git+https://github.com/keboola/python-docker-application.git@2.2.0 \
```
Comment (pivnicek) [inline Dockerfile:117]: GH has deprecated the `git` protocol

File: Dockerfile [hunk 3]
```diff
@@ -105,20 +105,27 @@ RUN mkdir /home/default \
...
         keboola.component \
-        chardet\<4 \
-        cryptography\<4 \
+        cryptography\<37 \
+        # additional torch packages
+        torch-scatter \
+        torch-sparse \
+        torch-cluster \
+        torch-spline-conv \
+        torch-geometric \
```
Comment (pivnicek) [inline Dockerfile]: these take a looong time to install

Comment (pivnicek) [reply Dockerfile]: so I removed them

File: Dockerfile (hunk 1 above)
Comment (pepamartinec) [reply Dockerfile]: after our discussion please try one more time with the opriginal packages (it installed python 2 packages, now it install python 3 packages)

**Comment by pivnicek** [comment]
@joker806 tak vypada ok,  muzu to releasnout?

**Review by pepamartinec — APPROVED** [review]
ok

**Pull request merged**